jersey-servers/src/main/java/com/palantir/remoting1/servers/jersey/TraceEnrichingFilter.java

```diff
@@ -50,13 +51,14 @@ public void filter(ContainerRequestContext requestContext) throws IOException {
         if (Strings.isNullOrEmpty(traceId)) {
             // HTTP request did not indicate a trace; initialize trace state and create a span.
             Tracer.initTrace(Optional.<Boolean>absent(), Tracers.randomId());
-            Tracer.startSpan(operation);
+            Tracer.startSpan(operation, SpanType.SERVER_INCOMING);
```

Comment:
This is currently captured when observing traces rather then when making them, see AsyncSlf4jSpanObserver constructors, for instance. Do you think the service name should live in static state instead? Is it different from span to span or is it part of the thread-local Trace?